pkg/monitor/cluster/cluster.go

```diff
@@ -67,8 +75,14 @@ func NewMonitor(env env.Interface, log *logrus.Entry, oc *api.OpenShiftCluster,
 func (mon *Monitor) Monitor(ctx context.Context) error {
 	mon.log.Debug("monitoring")
 
+	// Emit version information on the desired and actual cluster versions
+	err := mon.emitClusterVersion()
```

Comment:
This should follow emitAPIServerHealthzCode, so that we don't try to do this if the API server is down.

I think that Monitor() should not return an error, instead it should log errors returned by the sub-functions.

If the API server is not reachable, the function should return early.  Otherwise it should keep going.